examples/02_model_collaborative_filtering/multi_vae_deep_dive.ipynb

```diff
@@ -0,0 +1,1829 @@
+{
```

Comment:
<pre class="ql-syntax" spellcheck="false">Pandas version: 1.0.5 
Tensorflow version: 2.2.0-rc1 
Keras version: 2.3.1 
</pre><p>what would be the difference if you used the library versions of the repo?</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/microsoft/recommenders/pull/1191/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='3'/>

Reply:
We just realised that the model the notebook can be run with Pandas version: 0.25.3 and Tensorflow version: 1.15.2. So no difference on that.

We could not find a version of Keras used in the repo.